### PR TITLE
feat(metrics): export Filebeat metrics with beat-exporter

### PR DESCRIPTION
**Pull request by lichuan0620**
**What this PR does / why we need it**:

Integrates [beat-exporter](https://github.com/trustpilot/beat-exporter), an open-sourced Prometheus exporter that export the content of Filebeat statistics API as metrics.  

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: release/logging-filebeat.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ _config:
           memory: 100Mi
           cpu: 200m
       image: '[[ registry_release ]]/filebeat:[[ imageTagFromGitTag ]]'
-      imagePullPolicy: Always
+      imagePullPolicy: IfNotPresent
```
Comment (supereagle) [inline release/logging-filebeat.yaml]: It is required to be `Always`.

**Review by supereagle — COMMENTED** [review]
Generally LGTM

File: release/logging-filebeat.yaml (hunk 1 above)
Comment (lichuan0620) [reply release/logging-filebeat.yaml]: fixed

**Pull request merged**